File: TranformacionServer/TransformacionCliente.java
import java.net.*;
import java.io.*;
 
class cliente{
	DataInputStream input;
	BufferedInputStream bufferEntrada;
	BufferedOutputStream bufferSalida;
	int in;
	byte[] byteArray;
	final String cancion
	
	public static void main (String[] args){

		 
	}

	private static void run() {
	 	//Fichero a transferir
		 cancion = "/home/esmeralda/TequilaMusic/Repositorio/Los_Angeles_Azules_De_Plaza_en_Plaza/La_Cumbia_del_Infinito.mp3"f";
		 
		try{
			final File archivoEnviar = new File( cancion );
			Socket cliente = new Socket("localhost", 1234);
			bufferEntrada = new BufferedInputStream(new FileInputStream(archivoEnviar));
			bufferSalida = new BufferedOutputStream(cliente.getOutputStream());

			bufferSalida.write();
			//Enviamos el nombre del fichero
			DataOutputStream dataOutput=new DataOutputStream(cliente.getOutputStream());
			dataOutput.writeUTF(archivoEnviar.getName());
			//Enviamos el fichero
			byteArray = new byte[8192];
			while ((in = bufferEntrada.read(byteArray)) != -1){
			bufferSalida.write(byteArray,0,in); 
		}
		 
		bufferEntrada.close();
		bufferSalida.close();
		 
		}catch ( Exception e ) {
		 System.err.println(e);
	 	}
	 }
}

File: TranformacionServer/TransformacionCliente.py
#!/usr/bin/env python
 
#importamos el modulo para trabajar con sockets
import socket
from pydub import AudioSegment

HOST = 'localhost'
PORT = 1234
ADDR = (HOST,PORT)
#Creamos un objeto socket para el servidor. Podemos dejarlo sin parametros pero si 
#quieren pueden pasarlos de la manera server = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
client = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
client.connect(ADDR)
client.send("Los_Angeles_Azules/")
client.send("De_Plaza_en_Plaza/")

client.send("La_Cumbia_del_Infinito")

print "Enviando cancion"
sound = "/home/esmeralda/TequilaMusic/Repositorio/Los_Angeles_Azules_De_Plaza_en_Plaza/La_Cumbia_del_Infinito.mp3"
bytes = open(sound).read()
print len(bytes)
client.send(bytes)

#print 'ruta donde se guardo la cancion'
#print client.recv(1024)

client.close()

